src/main.js

```diff
@@ -90,7 +90,7 @@ class SliderEntityRow extends LitElement {
       ${(c.stateObj.state === "unavailable")
         ? html`
             <span class="state">
-            unavailable
+            ${this.hass.resources[this.hass.language]["state.default.unavailable"]}
```

Comment:
You can do even better with the provided translation function:
`${this.hass.localize("state.default.unavailable")}`

Reply:
Thanks! Let me test that...